server/apps/postgres-app/README.adoc

```diff
@@ -124,3 +124,11 @@ To run it, simply type:
 ....
 docker compose -f docker-compose-distributed.yml up -d
 ....
+
+== Administration Operations
+=== Clean up data
+
+To clean up some specific data, that will never be used again after a long time, you can execute the SQL queries `clean_up.sql`.
+The never used data are:
+- mailbox_change
+- email_change
```

Comment:
Should we add a simple cron setup instruction e.g. every day?

Reply:
I thought it was first, but IMO it is just a cleanup SQL.
 how to run it, when to run it, it depends on the Admin. Not specific is cron job